source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
@martndemus Babel's transpiler does `this._super(...arguments)` just fine:

* [using super and ...arguments](http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&playground=true&code=function%20bob()%20%7B%0A%20%20return%20this.fmt(...arguments)%3B%0A%7D)

The alternative introduces an array and a bunch of boilerplate:

* [using super and reset ..calledWithArgs](http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&playground=true&code=function%20bob(...bobArgs)%20%7B%0A%20%20return%20this.fmt(...bobArgs)%3B%0A%7D)

AirBnB's best practices are an interesting point to consider, but from a practical approach I disagree with the suggestion. super is an excellent place to use `...arguments`.

I would agree with their suggestion if you plan to manipulate arguments in any way though- then you should absolutely make an array instead of chaning the arguments object itself.